src/main/java/org/mskcc/limsrest/limsapi/cmoinfo/SampleTypeCorrectedCmoSampleIdGenerator.java

```diff
@@ -150,4 +152,8 @@ private void notifyAboutCorrectedCmoIdFailure(CorrectedCmoSampleView correctedCm
                     " sample: %s", correctedCmoSampleView.getId()));
         }
     }
+
+    private void validate(List<CorrectedCmoSampleView> views) {
```

Comment:
`validate` logic is left empty intentionally?

Reply:
Nope, I forgot to delete it. thanks!